### ISSUE TITLE
simple-installer: Uses Private IP Address (eth0) instead of Public

### ISSUE THREAD

**Issue by reillychase** [2600hz/community-scripts#69]
I am installing Kazoo on AWS. AWS servers are assigned a private ip address (172.31.x.x) on eth0, but are accessible from public IP. I finally found a CentOS 6.5 image to use, and was able to get the installer to work (after failed attempts at installing on RHEL 7.2 and CentOS 6.6)

Everything appears to have installed correctly, but the private IP was used for everything, making the monster-ui unaccessible.

I edited the index.html file and updated it with the public IP, replacing the private IP.

Now when I access the Public IP over HTTP, the monster-ui does load, but when I try to log in the request is being sent to the private IP Address of the server.

**Comment by swysor** [comment]
Yeah, I believe you will need to set URL in /var/www/html/monster-ui/js/config.js to the public IP or it won't work. I have not heard of people having much luck with this and AWS, their networking setup definitely makes for a difficult journey.

**Comment by reillychase** [comment]
Thanks @swysor I can definitely foresee more problems with AWS and NAT + SIP. Is there a recommend host for a Kazoo all-in-one server? Digital Ocean does not offer CentOS 6.5

**Comment by swysor** [comment]
It should work with the patch current Centos6 (6.7 now IIRC?). If it does not please let me know what you see failing and I can update the scripts to fix. I do intend on updating this whole thing to support for centos 7 and 4.0, but I have not had time to get that completed for the next little while.

**Comment by reillychase** [comment]
Ok thanks, I am attempting to install on DigitalOcean CentOS 6.8 now (thats the only 6 version available there). I'll let you know how it goes

**Comment by swysor** [comment]
Sounds good, thank you! Yeah, if it does not work, that's definitely a defect and I will fix it :D

**Comment by reillychase** [comment]
CentOS 6.8 Install went great on DigitalOcean, no problems looks like. HTTP for monster-ui is working, but I get 'invalid credentials' when logging in even though I know the info is correct. Let me know if you have any ideas

**Comment by reillychase** [comment]
Ah here it is:  

```
- Creating admin account with: sup crossbar_maintenance create_account "rchase" "sip.kazoo.rchase.com" "rchase" "my-p@ssw0rd"
## **failed**


 - setting up apps with: sup crossbar_maintenance init_apps /var/www/html/monster-ui/apps http://$system_ip_address:8000/v2
ok
```

**Comment by reillychase** [comment]
If I take that failed command and enter it myself, the output is:

failed to validate account properties(error): '{}'
failed

**Comment by reillychase** [comment]
Ok I found a work around, it didnt like the "Realm" param for some reason (sip.kazoo-install.example.com). After I tried shortening it to just 'sip.kazoo' then it created the account. I'm sure this will cause problems later on, any idea why that happened?

**Solution:** 
 sup crossbar_maintenance create_account "rchase" "sip.kazoo-install" "rchase" "my-p@ssw0rd"

**Comment by reillychase** [comment]
Now I am logged into the admin, so I tried switching the realm back to sip.kazoo-install.example.com, but received the error that it is already in use. So I guess the install script needs to be updated to pick a new realm for the admin user?

"message": "Account realm already in use",

**Comment by swysor** [comment]
if you recursively remove /srv/* you can rerun migrate, then account creation and init_apps, that will completely clean the db (make sure you stop whapps/ecallmgr and bigcouch after you do the srv remove. I would guess the regex that validates the realm is not liking the - since that's not found in the \w charset.

**Comment by swysor** [comment]
Also note for cloud servers that IPTables will need to be setup/started ASAP, due to the varying topologies we dont generate Iptables on the simple-installer (we do generate these via cluster manager but that's a completely different installer).

**Comment by swysor** [comment]
Oh the realm is auto-generated from the hostname actually 
https://github.com/2600hz/community-scripts/blob/master/simple-installer/setup_packages#L222
